doc/quickstart.md

```diff
@@ -0,0 +1,111 @@
+# vfkit quick start
+
+## Introduction
+
+vfkit is a macOS command line hypervisor.
+It uses [Apple's Virtualization Framework](https://developer.apple.com/documentation/virtualization?language=objc) to run virtual machines.
+You start a virtual machine by running vfkit with a set of arguments describing the virtual machine configuration/hardware.
+When vfkit stops, the virtual machine stops running.
+It runs on both x86_64 and aarch64 Macs.
+
+
+## Installation
+
+You can get vfkit either by downloading it from [its release page](https://github.com/crc-org/vfkit/releases), or get it from [brew](https://brew.sh/):
+```
+# Only the first time
+$ brew tap cfergeau/crc
+
+$ brew install vfkit
+```
+
+
+## Quick start
+
+### Getting a disk image
+
+Your virtual machine will need an operating system to run, so you need to download a disk image first.
+The image needs to be in the raw or iso format, qcow2 or VirtualBox images cannot be used by vfkit.
```

Comment:
This sentence is confusing. I believe the comma before `qcow` should be a full stop instead ,as both are not supported, right?